src/styles/product-card.css

```diff
@@ -0,0 +1,107 @@
+.product-card {
+  display: flex;
+  flex-direction: column;
+
+  width: 162px;
```

Comment:
Set 200 px here and use `box-sizing: border-box` instead of width calculating
Also this value is wrong, and I believe it can break tests (162 + 32 + 2 = 198px)

Reply:
Nice, thanks